lms/djangoapps/certificates/generation_handler.py

```diff
@@ -87,59 +156,68 @@ def can_generate_allowlist_certificate(user, course_key):
         # The invalidation list overrides the allowlist
-        log.info(
-            '{user} : {course} is on the certificate invalidation list. Certificate cannot be generated.'.format(
-                user=user.id,
-                course=course_key
-            ))
+        log.info(f'{user.id} : {course_key} is on the certificate invalidation list. Certificate cannot be generated.')
         return False
 
     enrollment_mode, __ = CourseEnrollment.enrollment_mode_for_user(user, course_key)
     if enrollment_mode is None:
-        log.info('{user} : {course} does not have an enrollment. Certificate cannot be generated.'.format(
-            user=user.id,
-            course=course_key
-        ))
+        log.info(f'{user.id} : {course_key} does not have an enrollment. Certificate cannot be generated.')
         return False
 
     if not IDVerificationService.user_is_verified(user):
         log.info(f'{user.id} does not have a verified id. Certificate cannot be generated.')
         return False
 
     if not _is_on_certificate_allowlist(user, course_key):
-        log.info('{user} : {course} is not on the certificate allowlist. Certificate cannot be generated.'.format(
-            user=user.id,
-            course=course_key
-        ))
+        log.info(f'{user.id} : {course_key} is not on the certificate allowlist. Certificate cannot be generated.')
         return False
 
-    log.info('{user} : {course} is on the certificate allowlist'.format(
-        user=user.id,
-        course=course_key
-    ))
+    log.info(f'{user.id} : {course_key} is on the certificate allowlist')
     cert = GeneratedCertificate.certificate_for_student(user, course_key)
     return _can_generate_allowlist_certificate_for_status(cert)
 
 
+def _can_generate_v2_certificate(user, course_key):
+    """
+    Check if a v2 course certificate can be generated (created if it doesn't already exist, or updated if it does
+    exist) for this user, in this course run.
+    """
+    if not _is_using_v2_course_certificates(course_key):
+        # This course run is not using the v2 course certificate feature
+        log.info(f'{course_key} is not using v2 course certificates. Certificate cannot be generated.')
+        return False
+
+    # TODO: Further implementation will be added in MICROBA-923
+    log.warning(f'Ignoring check on V2 course certificates for {user.id}: {course_key}')
+    return False
+
+
 def is_using_certificate_allowlist_and_is_on_allowlist(user, course_key):
     """
     Return True if both:
     1) the course run is using the allowlist, and
     2) if the user is on the allowlist for this course run
     """
-    return is_using_certificate_allowlist(course_key) and _is_on_certificate_allowlist(user, course_key)
+    return _is_using_certificate_allowlist(course_key) and _is_on_certificate_allowlist(user, course_key)
 
 
-def is_using_certificate_allowlist(course_key):
+def _is_using_certificate_allowlist(course_key):
```

Comment:
Goes with first comment: why are we designating this as private if we import it elsewhere? Can we just remove the `_` in both cases to follow convention?